ST/30_layered_architecture/README.md

```diff
@@ -0,0 +1,97 @@
+# Layered (N-tier) architecture
```

Comment:
Could you please rename README file to something more related for Layerd Architecture?
For example `Layerd_Architecture_README.md`

Reply:
Thanks, renamed